addons/im_livechat/static/src/new/core/message_model_patch.js

```diff
@@ -7,6 +7,6 @@ patch(Message.prototype, "im_livechat", {
         if (this.originThread.type !== "livechat") {
             return this._super();
         }
-        return this.author?.id !== this.originThread.operator?.id;
+        return !this.author?.id;
```

Comment:
This doesn't look correct: when the visitor is authenticated, this would wrongly flag that visitor as self author